api/pkg/service/catalog/catalog.go

```diff
@@ -139,11 +140,18 @@ func (s *service) CatalogError(ctx context.Context, p *catalog.CatalogErrorPaylo
 		}
 	}
 
+	var errorTypes []string
+
+	for key := range rs {
+		errorTypes = append(errorTypes, key)
+	}
+	sort.Strings(errorTypes)
```

Comment:
error types won't be changing with each call so we can defines file level array
```
var errorTypes = []String{
	parser.Info.String(), ...
}
```
and loop on this